### PR TITLE
Added initial playwright configuration

### PR DESCRIPTION
**Pull request by VaibhavMalik4187**
- Added basic playwright configuration in the playwright-action directory.
- Created a new playwright workflow to run playwright tests.

**Notes for Reviewers**

This PR fixes #

- [X] Yes, I signed my commits.

### CODE REVIEW

**Comment by VaibhavMalik4187** [comment]
@leecalcote Let's prototype the Playwright configuration here.

**Comment by VaibhavMalik4187** [comment]
The situation is very dicey here. Tests are randomly passing and failing. In the last CI run, every single test passed in the runner. All of them failed when I ran `npx playwright test` locally right after pushing the code.
image

File: .github/workflows/playwright.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+name: Playwright Tests
+on:
+  push:
+    branches: [main, master]
```
Comment (leecalcote) [inline .github/workflows/playwright.yml:4]: We don’t use main.

File: .github/workflows/playwright.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+jobs:
+  test:
+    timeout-minutes: 10
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+      - name: Install dependencies
```
Comment (leecalcote) [inline .github/workflows/playwright.yml:16]: Are there no playwright github actions?

File: .github/workflows/playwright.yml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+      - name: Run Playwright tests
+        run: |
+          cd playwright-action
+          npx playwright test
+      - uses: actions/upload-artifact@v3
+        if: always()
+        with:
+          name: /playwright-action/playwright-report
+          path: /playwright-action/playwright-report/
+          retention-days: 1
```
Comment (leecalcote) [inline .github/workflows/playwright.yml:33]: Will need a bump up.

File: .github/workflows/playwright.yml [hunk 4]
```diff
@@ -0,0 +1,33 @@
+name: Playwright Tests
+on:
+  push:
+    branches: [main, master]
+  pull_request:
+    branches: [main, master]
+jobs:
+  test:
+    timeout-minutes: 10
```
Comment (leecalcote) [inline .github/workflows/playwright.yml:9]: Not necessary and will only inevitably need a bump up.

File: playwright-action/playwright.config.js [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+
+  /* Configure projects for major browsers */
+  projects: [
+    {
+      name: "chromium",
+      use: { ...devices["Desktop Chrome"] },
+    },
+
+    {
+      name: "firefox",
```
Comment (leecalcote) [inline playwright-action/playwright.config.js:44]: Replace with Safari

File: playwright-action/tests/playground.spec.js [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+  await expect(page).toHaveTitle(
+    "Meshery Cloud | Cloud Native Management by Layer5"
+  );
+});
+
+test("Visit meshmap extension", async ({ page }) => {
+  await page.goto("https://playground.meshery.io/extension/meshmap");
+
+  // Expect a title "to contain" a substring.
+  await expect(page).toHaveTitle(
```
Comment (leecalcote) [inline playwright-action/tests/playground.spec.js:16]: This sort of thing is too fragile.

**Review by leecalcote — CHANGES_REQUESTED** [review]

**Review by leecalcote — APPROVED** [review]

**Pull request merged**